tests/framework/extensions/projects/projects.go

```diff
@@ -49,3 +51,19 @@ func GetProjectList(client *rancher.Client, clusterID string) (*management.Proje
 
 	return projectsList, nil
 }
+
+// ListProjectNames is a helper which returns a sorted list of project names
+func ListProjectNames(client *rancher.Client, clusterID string) ([]string, error) {
+	projectList, err := GetProjectList(client, clusterID)
+	if err != nil {
+		return nil, err
+	}
+
+	projectNames := make([]string, len(projectList.Data))
+
+	for idx, project := range projectList.Data {
+		projectNames[idx] = project.Name
+	}
+	sort.Strings(projectNames)
+	return projectNames, nil
+}
```

Comment:
Same for this one too

Reply:
I tried adding the List.names for this too but unfortunately we do not have the v1 schema for listing the projects in the downstream clusters. Its only applicable to the local cluster. Since, rbac runs majorly on the downstream clusters, leaving it as is for now.